post_assets.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import os
+import csv
+import django
+
+project_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(project_dir)
+django.setup()
+from core.models.asset import Asset, AssetModelNumber   # noqa
+
+
+def bulk_create_asset():
+    with open('assets.csv', 'r', ) as f:
+        skipped = dict()
+        skipped_records = []
+        inserted_records = []
+        data = csv.DictReader(f, delimiter='|')
```

Comment:
let's use `,` for the delimiter since we'll be getting the data from Google Spreadsheets (which uses commas by default)

Reply:
okay. Cool